cekit/builders/docker_builder.py

```diff
@@ -150,7 +147,7 @@ def _build_with_docker(self, docker_client):
                     "definitions."
                 )
 
-            raise CekitError(msg, ex)
+            raise CekitError(msg) from ex
```

Comment:
Didn't know about this construct!

Reply:
Realised the CEKitErrors weren't chaining the exceptions so looked up https://peps.python.org/pep-3134/ and https://blog.ram.rachum.com/post/621791438475296768/improving-python-exception-chaining-with